services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1323,7 +1323,7 @@ OFInstructionMeter installMeterForDefaultRule(IOFSwitch sw, long meterId, long r
     private OFInstructionMeter buildMeterInstruction(long meterId, IOFSwitch sw, OFFactory ofFactory,
```

Comment:
`buildMeterInstruction` must call `verifySwitchSupportsMeters` or must not be called if `verifySwitchSupportsMeters` does not confirmed meters support.
In other words we must check meters support in same way in all required points.

Reply:
That's great, but FL meters manipulation workflows refactoring isn't goal of this PR